packages/shared/locales/en.json

```diff
@@ -845,7 +845,9 @@
             },
             "transaction": {
                 "title": "Confirm transaction",
-                "info": "Confirm that the transaction information displayed on your Ledger device matches the information displayed below. If they match, press both buttons on your Ledger as prompted."
+                "info": "Confirm that the transaction information displayed on your Ledger device matches the information displayed below. If they match, press both buttons on your Ledger as prompted.",
+                "warning": "The transaction information displayed on your Ledger device won't match the information displayed below. The amounts displayed on the leger will differ as a result of minimum migratable amount limit on legacy network, press both buttons on your Ledger as prompted.",
```

Comment:
I would not make what shows ledger different from that we display in the transaction, we need to make a 100% exact match, but I would do it the other way around, explain that this is not the migratable amount, that less will be migrated but that we can have to crreate the tx with a min of 1 IOTA